### PR TITLE
Update system-requirements.md

### PR DESCRIPTION
**Pull request by JustinN-Replicated**
k8s gcr dependency for kurl

### CODE REVIEW

**Review by markpundsack — DISMISSED** [review]

File: src/markdown-pages/install-with-kurl/system-requirements.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ IP addresses for these services can be found in [replicatedhq/ips](https://githu
...
 | amazonaws.com | tar.gz packages are downloaded from Amazon S3 during embedded cluster installations. The IP ranges to allowlist for accessing these can be scraped dynamically from the [AWS IP Address](https://docs.aws.amazon.com/general/latest/gr/aws-ip-ranges.html#aws-ip-download) Ranges documentation. |
 | k8s.kurl.sh   | Kubernetes cluster installation scripts and artifacts are served from [kurl.sh](https://kurl.sh). Bash scripts and binary executables are served from kurl.sh. This domain is owned by Replicated, Inc which is headquartered in Los Angeles, CA.                                                |
+| k8s.gcr.io   | Kubernetes component images are downloaded from the Kubernetes registry on [GCR](https://cloud.google.com/container-registry). These components include the following: `kubeadm, kube-controller-manager, kube-scheduler, kube-proxy, pause, etcd, and coredns`. For more details on these Kubernetes tools, please visit the [Kubernetes documentation](https://kubernetes.io/docs/home/). |
```
[the remark this answers is not in the document]
Comment (JustinN-Replicated) [reply src/markdown-pages/install-with-kurl/system-requirements.md:46]: Makes sense, just changed it to be before k8s.kurl.sh in the table

**Comment by JustinN-Replicated** [comment]
tyty, committed those. lgtm

**Review by ajp-io — APPROVED** [review]

**Review by markpundsack — APPROVED** [review]
Looks good. The note above about where to find the IP addresses is no longer accurate as that link only covers Replicated-owned IP addresses.

Maybe we need to move the note to the k8s.kurl.sh section specifically. And then maybe the new k8s.gcr.io section needs a note about where to find the IP addresses?

**Comment by ajp-io** [comment]
@JustinN-Replicated Actually today is the day we start creating PRs in the new docs repo. It looks like this could go [here](https://docs.replicated.com/enterprise/installing-general-requirements/#kubernetes-version-compatibility) instead.

**Comment by JustinN-Replicated** [comment]
Going to merge this one for now; @ajp-io I am assuming the docs team will see it and can move to the new docs site.

**Pull request merged**